applications/asset_tracker_v2/src/cloud/cloud_codec/lwm2m/lwm2m_codec.c

```diff
@@ -158,9 +96,13 @@ static int object_path_list_add(struct cloud_codec_data *output, const char * co
 }
 
 /* Function that is called whenever the configuration object is written to. */
-static int config_update_cb(uint16_t obj_inst_id, uint16_t res_id, uint16_t res_inst_id,
+STATIC int config_update_cb(uint16_t obj_inst_id, uint16_t res_id, uint16_t res_inst_id,
 			    uint8_t *data, uint16_t data_len, bool last_block, size_t total_size)
 {
+	/* Since we are dependent on providing all configurations in the
+	 * CLOUD_CODEC_EVT_CONFIG_UPDATE event. All configuration are retrieved whenever the
+	 * configuration object changes.
+	 */
```

Comment:
```suggestion
	/* Because we are dependent on providing all configurations in the
	 * CLOUD_CODEC_EVT_CONFIG_UPDATE event, all configuration is retrieved whenever the
	 * configuration object changes.
	 */
```